_api-reference/script-apis/create-stored-script.md

```diff
@@ -44,7 +43,7 @@ All parameters are optional.
 | Field | Data Type | Description | 
 :--- | :--- | :---
 | lang | String | Scripting language. Required. |
-| source | String or Object | Required. <br />  <br /> For scripts, a string with the contents of the script. <br />  <br /> For search templates, an object that defines the search template. Supports the same parameters as the [Search API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/search)'s request body. Search templates also support Mustache variables. |
+| source | String or Object | Required. <br />  <br /> For scripts, a string with the contents of the script. <br />  <br /> For search templates, an object that defines the search template. Supports the same parameters as the [Search API]({{site.url}}{{site.baseurl}}/api-reference/search)'s request body. Search templates also support Mustache variables. |
```

Comment:
No need to use possessive form and the "API" should be broken out of the link since it isn't included in the linked header. Possibly could be reformatted as `[Search API Reference]({{site.url}}{{site.baseurl}}/api-reference/search)` depending on whether or not we decide to inherit ToC hierarchy in links (see an earlier comment that mentions this).

```
| source | String or Object | Required. <br /> <br /> For scripts, a string with the contents of the script. <br /> <br /> For search templates, an object that defines the search template. Supports the same parameters as the [Search]({{site.url}}{{site.baseurl}}/api-reference/search) API request body. Search templates also support Mustache variables. |
```